starter-code/src/viking.js

```diff
@@ -1,11 +1,104 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+Soldier.prototype.attack = function(){
+  return this.strength;
+}
+
+Soldier.prototype.receiveDamage = function(damage){
+
+ this.health -= damage;
+}
 
 // Viking
-function Viking() {}
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+Viking.prototype.receiveDamage = function(damage){
+
+  this.health -= damage;
+
+  if(this.health <= 0){
+      return this.name + " has died in act of combat";
+  } else { return this.name + " has received "+ damage +" points of damage"}
+}
+
+Viking.prototype.battleCry = function(){
+  return 'Odin Owns You All!'
+}
+
+function Viking(name, health, strength){
+  Soldier.call(this, health, strength);
+  this.name = name;
+}
```

Comment:
Respetad el orden siempre: primero `Soldier.call()` en la función constructora, segundo la creación del objeto en el prototipo del constructor y por último todas las funciones que queráis crear en su prototipo. No olvidéis el orden, es importante